dask_sql/physical/rel/custom/create_experiment.py

```diff
@@ -0,0 +1,207 @@
+import logging
+
+import dask.dataframe as dd
+import pandas as pd
+from dask_ml.wrappers import ParallelPostFit
+
+from dask_sql.datacontainer import ColumnContainer, DataContainer
+from dask_sql.java import org
+from dask_sql.physical.rel.base import BaseRelPlugin
+from dask_sql.utils import convert_sql_kwargs, import_class
+
+logger = logging.getLogger(__name__)
+
+
+class CreateExperimentPlugin(BaseRelPlugin):
```

Comment:
General comment (applies to this class and all the examples/documentation below): the model name is currently independent of the experiment name. Would you think it makes sense to change that, so that the best performing model has the same name as the experiment? From my view, that would make it easier for people to do the connection. Because now I fear that having multiple experiments will override the output model.
In any case, we should mention in the documentation somewhere how the resulting model is named.

Reply:
Hi @nils-braun, Thanks for taking the time to review the code and clearly explaining the bugs and typos.

I also like the idea of storing the best-performing model with the experiment name. and added the same in the documentation.

Please review and let me know if any further changes are needed